src/formatter/get_color_fns.ts

```diff
@@ -23,6 +23,7 @@ export default function getColorFns(enabled: boolean): IColorFns {
           [Status.PENDING]: colors.yellow.bind(colors),
           [Status.SKIPPED]: colors.cyan.bind(colors),
           [Status.UNDEFINED]: colors.yellow.bind(colors),
+          [Status.UNKNOWN]: colors.yellow.bind(colors),
```

Comment:
When is this used?

Reply:
Error with the `UNKNOWN` one removed:

```
src/formatter/get_color_fns.ts:19:16 - error TS7053: Element implicitly has an 'any' type because expression of type 'Status' can't be used to index type '{ [Status.AMBIGUOUS]: any; [Status.FAILED]: any; [Status.PASSED]: any; [Status.PENDING]: any; [Status.SKIPPED]: any; [Status.UNDEFINED]: any; }'.
  Property '[Status.UNKNOWN]' does not exist on type '{ [Status.AMBIGUOUS]: any; [Status.FAILED]: any; [Status.PASSED]: any; [Status.PENDING]: any; [Status.SKIPPED]: any; [Status.UNDEFINED]: any; }'.
```

I'm not sure if this function is ever called with `UNKNOWN`, but the compiler's point is that it _could_ happen at runtime, and since the function signature requires it to return an `IColorFn`, we have to cater to all values in the enum in our object.

(I keep running into smarts I didn't know TypeScript had here...)